hc/accounts/tests/test_switch_team.py

```diff
@@ -1,31 +1,66 @@
 from hc.test import BaseTestCase
 from hc.api.models import Check
+from django.urls import reverse
 
 
 class SwitchTeamTestCase(BaseTestCase):
 
-    def test_it_switches(self):
-        c = Check(user=self.alice, name="This belongs to Alice")
-        c.save()
+    def setUp(self):
+        super(SwitchTeamTestCase, self).setUp()
+        self.url = reverse("hc-switch-team", args=(self.alice.username, ))
+
+    def test_user_can_switch_team(self):
+        """
+            Test user can switch teams.
+
+            Bob is a member of Alice's team,
+            he is trying to swith to that team.
+            Asserts that he is successfully redirected to hc-checks.
+        """
+        check = Check(user=self.alice, name="This belongs to Alice")
+        check.save()
 
         self.client.login(username="bob@example.org", password="password")
 
-        url = "/accounts/switch_team/%s/" % self.alice.username
-        r = self.client.get(url, follow=True)
+        response = self.client.get(self.url, follow=True)
+
+        # Assert the contents of response
+        self.assertRedirects(response, reverse("hc-checks"))
```

Comment:
The task requires that you `check the contents for the response` NOT where the user is redirected